src/cmds/service.c

```diff
@@ -12,43 +12,68 @@
 #include <unistd.h>
 #include <stdlib.h>
 #include <mem/misc/pool.h>
-
 #include <util/dlist.h>
+#include <sys/types.h>
+#include <signal.h>
+
+#define len 32
```

Comment:
Our codestyle points that macros names must be in upper case
Use more informative  symbols names